src/validation/error.messages.ts

```diff
@@ -133,4 +134,5 @@ export enum ErrorMessages {
   ROLES_AND_RESPONSIBILITIES_INVALID_CHARACTERS = "Role and responsibilities must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
   CONTACT_NAME_INVALID_CHARACTERS = "Contact name must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
   EMAIL_INVALID_FORMAT = "Enter an email address in the correct format, like name@example.com",
+  INVALID_OE_LENGTH = "OE number must be 8 characters",
```

Comment:
Would suggest getting 'number' into the constant name (e.g. INVALID_OE_NUMBER_LENGTH) as an overseas entity doesn't really have a length.

Also, wondering if message is correct/accurate enough? Does this 'number' only contain numbers and no letters/special chars? If yes (just numbers), could it maybe say "OE number must be 8 digits"?

Reply:
@mwestacott , it would contain OE and the number ex.OE123456 , but will rename the constant to INVALID_OE_NUMBER_LENGTH

Reply:
👍 thanks for clarifying.

Reply:
Length can be only number number, it is not improve readabitlity